src/bo4e/bo/preisblattnetznutzung.py

```diff
@@ -0,0 +1,42 @@
+"""
+Contains PreisblattNetnutzung class and corresponding marshmallow schema for de-/serialization
+"""
+
+import attr
+from marshmallow_enum import EnumField  # type:ignore[import]
+
+from bo4e.bo.preisblatt import Preisblatt, PreisblattSchema
+from bo4e.enum.bilanzierungsmethode import Bilanzierungsmethode
+from bo4e.enum.botyp import BoTyp
+from bo4e.enum.kundengruppe import Kundengruppe
+from bo4e.enum.netzebene import Netzebene
+
+
+# pylint: disable=too-few-public-methods
+@attr.s(auto_attribs=True, kw_only=True)
+class PreisblattNetznutzung(Preisblatt):
+    """
+    Die Variante des Preisblattmodells zur Abbildung der Netznutzungspreise
+    """
+
+    bo_typ: BoTyp = attr.ib(default=BoTyp.PREISBLATTNETZNUTZUNG)
+    # required attributes (additional to those of Preisblatt)
+    #: Die Preise gelten für Marktlokationen der angebebenen Bilanzierungsmethode
+    bilanzierungsmethode: Bilanzierungsmethode = attr.ib(validator=attr.validators.instance_of(Bilanzierungsmethode))
```

Comment:
The [BO4E documentation](https://www.bo4e.de/dokumentation/geschaeftsobjekte/bo-preisblattnetznutzung) does not mention if this is a required or optional attribute?
How do you know?

Reply:
I just guessed.

Reply:
#330